scripts/docker-release/push_docker.sh

```diff
@@ -20,48 +20,24 @@ readonly RETRIES=3
 # creating a Docker image to push. If this script does not detect an image
 # to be uploaded, it will fail.
 
-# This script is intended to be run from a CI job and will not work if run in
-# standalone manner unless certain envrionment variables are set.
-
 # Grab the tag we are working with
 
-echo "INFO: Determining latest tag"
-if [ ! -z ${TAG_NAME+x} ]
-then
-  echo "INFO: Detected TAG_NAME variable. Probably a Jenkins instance."
-  readonly GIT_TAG_DEFAULT=$(echo $TAG_NAME|sed s/^v//)
-else
-  echo "INFO: Did not detect TAG_NAME. Examining git log for latest tag"
-  readonly GIT_TAG_DEFAULT=$(git describe --abbrev=0|sed s/^v//)
-fi
-
-readonly CUR_TAG=${CUR_TAG:-$GIT_TAG_DEFAULT}
+readonly RELEASE_VERSION=${1}
 readonly DOCKER_REGISTRY_URL="docker.elastic.co"
 readonly DOCKER_IMAGE_NAME="observability/apm-agent-java"
-readonly DOCKER_PUSH_IMAGE="$DOCKER_REGISTRY_URL/$DOCKER_IMAGE_NAME:$CUR_TAG"
+readonly DOCKER_PUSH_IMAGE="$DOCKER_REGISTRY_URL/$DOCKER_IMAGE_NAME:$RELEASE_VERSION"
 readonly DOCKER_PUSH_IMAGE_LATEST="$DOCKER_REGISTRY_URL/$DOCKER_IMAGE_NAME:latest"
 
 # Proceed with pushing to the registry
 echo "INFO: Pushing image $DOCKER_PUSH_IMAGE to $DOCKER_REGISTRY_URL"
 
-if [ ${WORKERS+x} ]  # We are on a CI worker
-then
-  retry $RETRIES docker push $DOCKER_PUSH_IMAGE || echo "Push failed after $RETRIES retries"
-else  # We are in a local (non-CI) environment
-  docker push $DOCKER_PUSH_IMAGE || { echo "You may need to run 'docker login' first and then re-run this script"; exit 1; }
-fi
+docker push $DOCKER_PUSH_IMAGE || { echo "You may need to run 'docker login' first and then re-run this script"; exit 1; }
 
 readonly LATEST_TAG=$(git tag --list --sort=version:refname "v*" | grep -v RC | sed s/^v// | tail -n 1)
```

Comment:
[nit] this one is actually the "latest version", not the "latest tag".

Reply:
Do you want me to change it?